src/main/java/frc/robot/Logger.java

```diff
@@ -53,16 +55,18 @@ public static String getLogMsg(String logPath, String thisLogLevel, String messa
      */
     public void logLevel(String level, String message){
         List<String> levels = Arrays.asList(supportedLevels);
-        String minLevel = Config.getInstance().cfg.getString("log__" + logPath.toUpperCase());
+        String minLevel = Config.getInstance().cfg.getString("LOG__" + logPath.toUpperCase());
 
         // if logging is enabled at all for this logger,
         // and the level is recognized,
         // and our level is higher-up or equal to the minimum specified,
         // then log it
         if(minLevel != null && minLevel != "OFF" && levels.contains(level) && levels.indexOf(level) <= levels.indexOf(minLevel)){
             // if the level we're logging at is WARN or ERR, then log to STDERR, otherwise log to STDOUT
-            if(level == "WARN" || level == "ERR"){
-                System.err.println(getLogMsg(logPath, level, message));
+            if(level == "ERROR"){
```

Comment:
Level should be an enumeration so problems are caught at compile time rather than runtime.